submitted_models/marble_husky_sensor_config_1/launch/vehicle_topics.launch

```diff
@@ -0,0 +1,209 @@
+<?xml version="1.0"?>
+<launch>
+  <arg name="world_name"/>
+  <arg name="name"/>
+  <arg name="link_name" value="base_link"/>
+  <!--<arg name="pan_joint_name" value="pan_gimbal_joint" />
+  <arg name="tilt_joint_name" value="tilt_gimbal_joint"/>-->
+  <arg name="sensor_prefix" value="/world/$(arg world_name)/model/$(arg name)/link/$(arg link_name)/sensor"/>
+  <arg name="gimbal_sensor_prefix" value="/world/$(arg world_name)/model/$(arg name)/link/tilt_gimbal_link/sensor"/>
+  <group ns="$(arg name)">
+    <!--Create multipe bridges so that it can run in parallel-->
+    <node
+      pkg="ros_ign_bridge"
+      type="parameter_bridge"
+      name="ros_ign_bridge_pose"
+      args="/model/$(arg name)/pose@tf2_msgs/TFMessage[ignition.msgs.Pose_V">
```

Comment:
I'm observing some errors when running your bridge:

```
[ERROR] [1592228288.828625446]: Failed to create a bridge for topic [/model/X2/pose] with ROS type [tf2_msgs/TFMessage] and Ignition Transport type [ignition.msgs.Pose_V]
```
I suspect your version of the `ros_ign` bridge doesn't include this change:
https://github.com/ignitionrobotics/ros_ign/pull/67

Could you please double-check?

Reply:
I will check it out.

Reply:
I'll go ahead and update our ros_ign bridge by merging in [ignitionrobotics/ros_ign#67](url) and other changes that have been made since we forked the repo

Reply:
Also on the subject, we made our fork to add support for conversions from std_msgs/float64 to ignition doubles. Has the PR been approved? Because if so our custom version is no longer needed.

Reply:
> Also on the subject, we made our fork to add support for conversions from std_msgs/float64 to ignition doubles. Has the PR been approved? Because if so our custom version is no longer needed.

It looks merged: https://github.com/ignitionrobotics/ros_ign/pull/64

Reply:
Yes, I will update those instructions.

Reply:
It's merged, but I believe it's not part of the ros package yet (the binaries). Or it wasn't a month ago so from my understanding this bridge still needs to be built from source so include this change correct?

Reply:
`0.9.3` has been released: http://packages.osrfoundation.org/gazebo/ubuntu-stable/pool/main/r/ros-melodic-ros-ign-bridge/

We shouldn't need to compile the bridge from source.